### DISCUSSION TITLE
nuget.org tab not showing nuke?

### DISCUSSION THREAD

**Discussion by sheldonhull** [nuke-build/nuke#1315] [Q&A] [answered]
I read through nuget.org issue and also instructions on page and don't see the `nuke` instructions on the page displayed. Did something happen in a later issue I missed that removed this from showing up on nuget's page? 
![CleanShot 2024-01-22 at 20 28 59@2x](https://github.com/nuke-build/nuke/assets/3526320/26db8144-860c-43de-acb8-65d25e6a03e9)

related issue: https://github.com/NuGet/Home/issues/10784

**Comment by matkoch** [comment, marked as answer]
![image](https://github.com/nuke-build/nuke/assets/5005566/19dd7904-eea3-4f5b-80bc-49335d24d086)

**Reply by sheldonhull** [reply]
thank you, i missed that :-) Cheers
